README.md

```diff
@@ -1,2 +1,213 @@
+> functionality while deferring, as much as possible, additional capabilities
+> and features to add-on modules or applications.
+
+As above, the "small core" philosophy applies as much to the Governance model
+as it does to the code. At no point should the Foundation and the project
+leadership assume that "it knows best" when it comes to understanding what is
+best for the community as a whole.
+
+> In order to best serve the Node.js Community as a whole, the development,
+> release and issue management processes adopted by the Node.js Foundation
+> TSC must reflect these guiding principles.
+
+Likewise, should the merger proceed, the structure and governance of the new
+Foundation must first and foremost reflect the needs of the community,
+addressing real problems faced by that community.
+
+## What Problems Are We Trying To Solve?
+
+TBD
+
+## What Additional Goals Do We Have?
+
+TBD
+
+## What Concerns Have Been Raised?
+
+The following generalized concerns have been raised and must be dealt with as
+the consideration and discussion around the merger progresses:
+
+* *Lack of transparency in the process* ... Multiple technical project members
+  have expressed concerns about what they perceive as a lack of transparency
+  in the discussions around a merger.
+* *Lack of engagement among projects* ... Multiple technical project members
+  have expressed that they either have not been made aware of the discussions
+  or do not feel empowered to participate in the discussions around the
+  possible merger. Several have opted to disengage from the discussions
+  entirely due to a feeling that their concerns and objections will be ignored.
+* *Unclear Goals* ... Multiple technical project members have expressed concerns
+  that the goals of the proposed merger are unclear, specifically in terms of
+  how such a merger would be of benefit to the various technical projects.
+* *Unclear Timeline Motivations*... Mutiple technical project members have
+  expressed concerns about the timeline that has been proposed for the execution
+  of the proposed merger. For some, the process has felt "rushed" with little to
+  no clear reason given as to why.
+* *Disagreement with stated goals* ... Multiple technical project members have
+  expressed clear disagreement with the stated goals of the proposed merger.
+  Specifically, there has been objection to the idea that a joint umbrella
+  organization would be in the best interests of the Node.js Project.
+* *Possible loss of focus* ... Multiple technical project members have
+  expressed concerns that an umbrella organization will be inadequately
+  capable of focusing on the specific needs of individual projects. Several
+  have pointed at various dysfunctional aspects of the current JS Foundation
+  organizational and operational struture as evidence of that concern.
+
+## Foundation Structure Proposals
+
+TBD
+
+### Draft Foundation Mission Statements
```

Comment:
Do we want to document the draft Values / Mission statement the boards used as a guiding principal here? I think it is useful historical information to document.

Reply:
I think moving the concerns out to a separate doc with a link here would be good so that one can land before the other and vice versa.

Reply:
At this point, I'd prefer this document to be more the voice/view of the projects than that of the board. It would likely be useful to document the board's view of the values/mission in a separate background document.

Reply:
I'll update my PR that includes mission / vision to be a historical document instead